designs/developer/helm-45/helm-notes.md

```diff
@@ -0,0 +1,14 @@
+---
+parent: Developer
+version: 4.5
+---
+
+# Helm Release Notes
+After successful installation of a helm chart, users are directed to the topology view where they could see it visualized along with its components. On the topology view, the helm chart comes selected by default with the side panel showing release notes(from the associated Notes.txt file) for the respective helm chart.
```

Comment:
"where they could see it" could you update that to something "where they can see the associated Helm Release Notes"

Reply:
Changed